datasets/irish/metadata.yaml

```
@@ -1,24 +1,40 @@
 #Generated automatically by pmlb/write_metadata.py
+#Reviewed by Daniel Goldberg
 dataset: irish
-description: None yet. See our contributing guide to help us add one.
-source: None yet. See our contributing guide to help us add one.
-publication: None yet. See our contributing guide to help us add one.
+description: Educational transitions of Irish schoolchildren, aged 11 in 1967.
+source: https://www.openml.org/d/451
+publication: Monitoring the Learning Outcomes of Education Systems by Vincent Greaney and Thomas Kelleghan
 task: classification
 target:
   type: binary
-  description: None yet. See our contributing guide to help us add one.
-  code: None yet. See our contributing guide to help us add one.
+  description: Leaving certificate taken
+  code: 'not_taken'=1, 'taken'=2
 features: # list of features in the dataset
   - name: Sex
     type: binary
-    description: null # optional but recommended, what the feature measures/indicates, unit
-    code: null # optional, coding information, e.g., Control = 0, Case = 1
+    description: Sex of child # optional but recommended, what the feature measures/indicates, unit
+    code: 'male'=1, 'female'=0 # optional, coding information, e.g., Control = 0, Case = 1
     transform: ~ # optional, any transformation performed on the feature, e.g., log scaled
   - name: DVRT
-    type: continuous
+    type: ordinal
+    description: Drumcondra Verbal Reasoning Test score
   - name: Educational_level
     type: discrete
+    description: attained education level
+    code: >
+      '3rd_level_complete'=0, '3rd_level_incomplete'=1, '?'=2, 'Junior_cycle_incomplete-secondary_school'=3, 
+      'Junior_cycle_incomplete-vocational_school'=4, 'Junior_cycle_terminal_leaver-secondary_school'=5,
+      'Junior_cycle_terminal_leaver-vocational_school'=6, 'Primary_terminal_leaver'=7,
+      'Senior_cycle_incomplete-secondary_school'=8, 'Senior_cycle_incomplete-vocational_school'=9,
+      'Senior_cycle_terminal_leaver-secondary_school'=10
   - name: Prestige_score
     type: discrete
```

```suggestion
    type: ordinal
```

Maybe? Was there an order in prestige score calculated by Raftery and Hout, 1985?

---

Yes, higher values correlated to higher prestige. Ordinal makes sense.